src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -339,7 +339,7 @@
     <value>Upgrade MSBuildWorkspace</value>
   </data>
   <data name="CompareSymbolsCorrectlyDescription" xml:space="preserve">
-    <value>Symbols should be compared for equality, not identity.</value>
+    <value>Symbols should be compared for equality with an EqualityComparer.</value>
```

Comment:
I think we should revert the message given `EqualityComparer` is only available on most recent Roslyn bits. Instead, we should just update the code fix provider associated with this diagnostic to use the EqualityComparer when the comparer is available.